moovingon_ai/README.md

```diff
@@ -0,0 +1,42 @@
+# moovingon.ai
+
+## Overview
+[moovingon.ai](http://moovingon.ai/) offers a platform to help organizations manage their 24/7 operations using automated processes and centralized monitoring management. The platform includes alert aggregation and correlation, automatically linked with runbooks and an analytics engine for reports and dashboards.
+
+This integration will enable you to:
+
+1. Consolidate all your datadog alerts under a centralized platform.
+2. Automatically link incident remediation runbooks with alerts. 
+3. Log all remediation activities in datadog events. 
```

Comment:
The overview sounds like moovingon is only pulling data out of Datadog. Can you include what this integration is monitoring, what is being sent to Datadog, and the value?

Reply:
Consolidate all your datadog alerts under a centralized platform.
- is this ocnsolidating moovingon alerts within Datadog? 

Automatically link incident remediation runbooks with alerts. 
- Where is this linked? is it moovingon incidents or Datadogs? Is it Datadog alerts or moovingon? 

Log all remediation activities in datadog events. 
- what does this mean? what are the remediation activities?

Reply:
This platform was made for NOC. Every manual or automated action that we run from moovingon.ai will send an audit to datadog events.

The result is that lets say if I run a webhook from my platfrom, and that webhook restart the host of the monitor I'm corrently look at in DataDog, your users will be able to see the audits of the actions we take from my end.

ex.
1. Datadog triggered alert.
2. Event is being created in moovingon.ai
3. NOC engineer is taking a manual action, lets say restart the host manual OR automated flow is running, one of the steps is to restart the host by running a webhook / jenkins action.
4. The audit of the action will be send back to the Datadog monitor as an event so the user could correlate the behavior of the test in Datadog based on the action we made in our end. (ex. the alert is showing red for 5 mins because of a restart action from moovingon.ai, without this info you won't know why that datadog monitor is failing)

Reply:
Thanks for the insight! Can you add specifically what your product is doing like the above then what is being sent to Datadog so it's clear to users what's being sent to Datadog and what is being done within your product? The overview should be clear enough for someone who has no knowledge of our product and learn what your product does, what the integration does, what your integration specifically monitors, what is sent to Datadog.

Reply:
We've made some changes like you requested.